generator/java-exec/src/main/java/io/fabric8/maven/generator/javaexec/JavaExecGenerator.java

```diff
@@ -16,10 +16,7 @@
 package io.fabric8.maven.generator.javaexec;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
What's the purpose of this change? I usually avoid using wildcard imports since they clutter your namespace.

Reply:
My IDE must have done it and I missed to check that. Fixing right away.